resources/public/api/conf/1.0/retrieveBalance.raml

```diff
@@ -5,10 +5,25 @@ is:
   - errors.clientOrAgentNotAuthorised
   - errors.notFound
 displayName: "Retrieve a Self Assessment Balance [test only]"
+description: This endpoint allows a developer to retrieve the overall liability broken down into overdue, currently due (payable) and pending (not yet due) amounts. A unique identifier (National Insurance Number) for the account must be used.
+(annotations.sandboxData): !include scenarios/retrieveBalance.md
 (annotations.scope): "read:self-assessment"
 securedBy: [ sec.oauth_2_0: { scopes: [ "read:self-assessment" ] } ]
 responses:
   200:
+    body:
+      application/json:
+        type: !include schemas/retrieve_balance_response.json
+        examples:
+          example-1:
+            description: Example response
+            value: !include examples/retrieveBalance/retrieve_response.json
+          example-2:
+            description: Example response where nothing is due
```

Comment:
Maybe change this to something like: 
```suggestion
            description: Example response with nothing due
```
to be a bit more consistent with our other endpoints?